src/components/Registration.js

```diff
@@ -3,41 +3,45 @@ import "../scss/Registration.scss";
 import { QuizContext } from "../QuizContext";
 
 const Registration = ({quizStatus}) => {
-    const {setRegisteredUser, registeredUser, player, generateNewAvatar} = useContext(QuizContext);
+  const {setRegisteredUser, registeredUser, player, generateNewAvatar} = useContext(QuizContext);
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        setRegisteredUser(e.target.name.value);
-    };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setRegisteredUser(e.target.name.value);
+  };
 
-    return (
-        <div className="registration">
-            <h2>Zaraz zaczynamy</h2>
-            <img src={player.avatarUrl}/>
-            <button className="registration__shuffle-button" onClick={generateNewAvatar}>Wylosuj nowy avatar</button>
-            {!registeredUser && <Form handleSubmit={handleSubmit}/>}
-        </div>
-    );
+  return (
+    <div className="registration">
+      <h2>Zaraz zaczynamy</h2>
+      <img src={player.avatarUrl} />
+      {!registeredUser && <React.Fragment>
```

Comment:
w sumie tu by lepiej wyglądał ternary operator zamiast dwóch ifów, ale tak czy siak działa, więc to moze być trochę moje czepialstwo 😄

Reply:
`<>` can be used instead of `React.Fragment`